use_dynamic_obstacle: false
replan_trigger_obstacle: true
replan_trigger_interval: 5